### PR TITLE
Adding message incrementing functionality

### PR DESCRIPTION
**Pull request by jwholmes**
- Added `messages.js` file
 - Increments the `Messages this month` field by 1 for every message a user sends regardless of server
- Updated `index.js`
 - This update makes sure that the bot listens for new messages

### CODE REVIEW

File: intents/messages.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+const { fetchAirtableRecord } = require('./utils');
+
+const base = new Airtable({ apiKey: process.env.AIRTABLE_API_KEY }).base(
+    'appTigRgtCa5YFAQr'
+);
+
+async function incrementMessageCount(member) {
+    const existingUser = await fetchAirtableRecord(base, member.author.id);
+    const incrementedMessageCount = existingUser.get('Messages this month') + 1
+    console.log(existingUser.get('Messages this month'))
```
Comment (jonomuller) [inline intents/messages.js:13]: Debug statement - remove?

**Review by jonomuller — APPROVED** [review]
Looks good! My only question is that the monthly part of the `Messages this month ` field isn't really utilised. Will the field not be incremented over the lifetime of a user instead of the past month?

File: intents/messages.js (hunk 1 above)
Comment (jwholmes) [reply intents/messages.js:13]: Yep, I'll remove 🧹

**Comment by jwholmes** [comment]
@jonomuller 

> Will the field not be incremented over the lifetime of a user instead of the past month?

Should have mentioned this in the PR, but this is handled within Airtable.

The `Messages this month` field will be zeroed in the early hours of the 1st of each month.

Screenshot 2021-09-06 at 13 45 22

**Pull request merged**